docs/CreateAKSClusterForScalarDB.md

```diff
@@ -21,32 +21,38 @@ When deploying ScalarDB Server, you must:
 
 The following are some recommendations for deploying ScalarDB Server. These recommendations are not required, so you can choose whether or not to apply these recommendations based on your needs.
 
+### Create at least three worker nodes and three pods
+
+To ensure that the AKS cluster has high availability, you should use at least three worker nodes and deploy at least three pods spread across the worker nodes. You can see the [sample configurations](../conf/scalardb-custom-values.yaml) of `podAntiAffinity` for making three pods spread across the worker nodes.
+
+{% capture notice--info %}
+**Note**
+
+If you place the worker nodes in different [availability zones](https://learn.microsoft.com/en-us/azure/availability-zones/az-overview) (AZs), you can withstand an AZ failure.
+{% endcapture %}
+
+<div class="notice--info">{{ notice--info | markdownify }}</div>
+
 ### Use 4vCPU / 8GB memory nodes for the worker node in the ScalarDB Server node pool
 
-From the perspective of commercial licenses, resources for one pod running ScalarDB Server are limited to 2vCPU / 4GB memory. In addition, we recommend deploying one ScalarDB Server pod and one Envoy pod on one worker node.
+From the perspective of commercial licenses, resources for one pod running ScalarDB Server  are limited to 2vCPU / 4GB memory. In addition, some pods other than ScalarDB Server pods exist on the worker nodes.
```

Comment:
```In addition, some pods other than ScalarDB Server pods exist on the worker nodes.```
sounds like some other pods **always** exist on the worker node, but the following sentence says other components **could** run on the worker node, so I felt a bit inconsistency between them.
I might miss something though.

Reply:
Ah, I see. You are right. It's not consistent.
I overlooked it.
I fixed it at a314ea5aad9d1db67fefbc1b7d6bcc9aec7515dd.

(I will update the same thing on other documents in another PR.)